libse/SubtitleFormats/UnknownSubtitle36.cs

```diff
@@ -28,7 +28,7 @@ namespace Nikse.SubtitleEdit.Core.SubtitleFormats
 
     public class UnknownSubtitle36 : SubtitleFormat
     {
-        private static readonly Regex RegexTimeCodes = new Regex(@"^\d+:\d+:\d+:\d+           ,           \d+:\d+:\d+:\d+           ,.*$", RegexOptions.Compiled);

```

Comment:
I think `.*` is not a good idea.
we only need to filter time-code with regex and the rest will be treated as string